docs/chapters/traversal/index.md

```diff
@@ -0,0 +1,178 @@
+# Understand Jina Recursive Document Representation
+
+In Jina, each `Document` is represented as a recursive representation (tree).
+
+A rooted recursive representation has a root node and every node has X children.
+In Jina, the root node is the document itself, while the *left* & *right* children are referred to as *chunks* and *matches* respectively.
+
+![rooted-binary-tree](img/rooted-binary-tree.png)
+
+The above image illustrates a minimal document structure: A document (root node) consists of two child nodes: *chunks* and *matches*.
```

Comment:
two `children`

Reply:
A minimal document structure is just a root node. Perhaps rephrase to

"The above image illustrates a basic document structure: A document (root node) and the two possible child nodes: *chunks* and *matches*.

Reply:
updated